src/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -380,7 +380,7 @@ public void DiagnosticSourceStartStop()
                     // Confirm that duration is set. 
                     Assert.NotNull(observer.Activity);
                     Assert.True(TimeSpan.Zero < observer.Activity.Duration);
-                    Assert.True(observer.Activity.StartTimeUtc + observer.Activity.Duration < DateTime.UtcNow);
+                    Assert.True(observer.Activity.StartTimeUtc + observer.Activity.Duration <= DateTime.UtcNow.AddTicks(1));
```

Comment:
Is the AddTicks(1) because https://github.com/dotnet/corefx/pull/16908 made it so that if Duration is 0 we bump it up to 1?